CRM/Price/Form/Preview.php

```diff
@@ -132,7 +132,9 @@ public function buildQuickForm() {
         foreach ($group['fields'] as $field) {
           $fieldId = $field['id'];
           $elementName = 'price_' . $fieldId;
-          CRM_Price_BAO_PriceField::addQuickFormElement($this, $elementName, $fieldId, FALSE, $field['is_required']);
+          if (!empty($field['options'])) {
+            CRM_Price_BAO_PriceField::addQuickFormElement($this, $elementName, $fieldId, FALSE, $field['is_required']);
+          }
```

Comment:
OK, so this is when FT of the price field is disabled. You can see in the field list, but Preview link shows empty.

<img width="695" alt="image" src="https://cloud.githubusercontent.com/assets/5929648/24690335/ebfef25e-19e9-11e7-80cb-9531ce6c6bf2.png">

Preview -

<img width="507" alt="image" src="https://cloud.githubusercontent.com/assets/5929648/24690346/fd1504c0-19e9-11e7-8608-467f258c0dee.png">

Not sure if this is the expected behavior?